src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -49,12 +45,6 @@ public static SbomFile ToSbomFile(this SPDXFile spdxFile)
     /// <returns></returns>
     public static SbomPackage ToSbomPackage(this SPDXPackage spdxPackage)
     {
-        if (spdxPackage.FilesAnalyzed &&
-            (spdxPackage.LicenseInfoFromFiles == null || !spdxPackage.LicenseInfoFromFiles.Any()))
-        {
-            throw new ParserException("Package license list was empty.");
```

Comment:
Before this change, code coverage shows that we're covering this throw. Does that mean that we need to update a unit test?

Reply:
Your review was too quick for me! :) Just added the test changes